src/main/app/idam-api/IdamAPI.ts

```diff
@@ -20,4 +20,27 @@ export class IdamAPI {
         return [];
       });
   }
+
+  public getUserById(id: string): Promise<User> {
```

Comment:
Do we need this method? We have the getUserDetails() method which calls out to the IDAM API to either retrieve email, ID or SSO ID.

Reply:
As discussed, will address this in a future PR :)